app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -541,4 +555,8 @@ class PatientSummaryScreenControllerTest {
 
     uiEvents.onNext(ScreenCreated())
   }
+
+  private fun startMobiusLoop() {
```

Comment:
Are we using this method somewhere or is it just placeholder until future commits?

Reply:
As the migration proceeds, calls to `setupController` will be replaced with calls to `startMobiusLoop`. Once the refactoring is complete, we will clean this up.